### ISSUE TITLE
scene graph generation for custom image

### ISSUE THREAD

**Issue by liuJP2** [waxnkw/IETrans-SGG.pytorch#21]
Hello~This is a really cool work! 
I'm trying to use SGDET to generate a scenegraph of my own image, I just need to modify the visualization/demo_imgs/ section in "bash cmds/50/transformer/demo_sgdet.sh visualization/demo_imgs/", I downloaded the transformer_sgdet_I70_E100.pth model.
I mean can this model automatically extract bounding box according to image? I just need to provide the image, right?

**Comment by liuJP2** [comment]
> 

And the visualization results given by vis.ipynb are the result of which model?

**Comment by waxnkw** [comment]
Yes, you just need to provide the image. The visualization results are of [vg1800_motif_sgcls_I90_E100.pth](https://thunlp.oss-cn-qingdao.aliyuncs.com/ietrans/vg1800_motif_sgcls_I90_E100.pth). The corresponding command is:
```shell
bash cmds/1000/motif/demo_sgdet.sh visualization/demo_imgs/
```

**Comment by liuJP2** [comment]
> Yes, you just need to provide the image. The visualization results are of [vg1800_motif_sgcls_I90_E100.pth](https://thunlp.oss-cn-qingdao.aliyuncs.com/ietrans/vg1800_motif_sgcls_I90_E100.pth). The corresponding command is:
> 
> ```shell
> bash cmds/1000/motif/demo_sgdet.sh visualization/demo_imgs/
> ```

Thank you for your reply
I have another question, for the demo_bbox.pk file, is it necessary to contain the following form of information?
['bowl', 'bowl', 'bowl', 'plate', 'carrot', 'carrot', 'carrot', 'carrot', 'carrot', 'carrot', 'carrot', 'table', 'cucumber', 'bowl', 'carrot', 'bag', 'carrot', 'spoon', 'food', 'carrot', 'carrot', 'carrot', 'tomato', 'plate', 'carrot', 'bag', 'broccoli', 'food', 'broccoli', 'spoon', 'handle', 'food', 'vegetable', 'food', 'plastic', 'carrot', 'fork', 'carrot', 'countertop', 'wrapper', 'pepper', 'carrot', 'plastic', 'countertop', 'cucumber', 'carrot']]
Does that mean it has to be like this：
{
    image_name1: [
        np.ndarray([[x1, y1, x2, y2], ... ]),
        [label1, label2, ...]
    ],
}
If I only have np.ndarray, is that allowed? That is, modify the format as follows:
{
    image_name1: [
        np.ndarray([[x1, y1, x2, y2], ... ])
    ],
}
